README.md

```diff
@@ -1,3 +1,107 @@
 # Argyle iOS SDK example project
 
-Documentation can be found [here](https://argyle.io/docs/argyle-link/ios-sdk-integration)
+Argyle Link iOS SDK provides a way to integrate [Argyle Link](https://argyle.io/docs/argyle-link) into your iOS app.
+
+If you are looking to update Argyle Link to the newest version, navigate to [upgrade guide](https://github.com/argyle-systems/argyle-link-ios/blob/master/UPGRADING.md).
+
+**Requirements:**
+- iOS 12.0+
+- Xcode 13.0+
+- Swift 5+ 
+  
+**Note:** We recommend you to lock your app to portrait orientation.
+
+---
+
+### 1. Adding the SDK dependency
+
+**CocoaPods**
+
+[CocoaPods](https://cocoapods.org/) is a dependency manager for Cocoa projects. For usage and installation instructions, visit their website. To integrate Argyle into your Xcode project using CocoaPods, specify it in your `Podfile:`
+
+`pod 'Argyle', '3.x.x'`
```

Comment:
Shouldn't we be providing docs for `4.x.x` version? Am I missing something?

Reply:
Thanks, for noticing. Indeed